package.json

```diff
@@ -9,18 +9,20 @@
     "src"
   ],
   "scripts": {
+    "#postinstall": "postinstall hook is required to 'fix' ava d.ts file",
+    "postinstall": "rimraf node_modules/ava/index.d.ts",
```

Comment:
Neat idea with the comment for postinstall. You could rephrase to:
"Remove bundled typings to use patched version with Promise support"